### PR TITLE
[cli][m] removes old paster cli commands 

### LINKED ISSUE #5110: Remove old paster CLI commands
Once #5108 and #5109 are done we can remove the old paster commands in `ckan/lib/cli.py`.

At least for CKAN 2.9 let's remove the actual commands only and keep the `CkanCommand` class as this is present in the toolkit and needed by extensions to implement paster-based commands. Also utilities like `load_config` as they might be imported from extensions.

There's is some click-based stuff in there which I don't know if it's still relevant (`click_config_option`, `paster_click_group`...). If no longer needed it can be removed.

### PR DESCRIPTION
**Pull request by EvgeniiaVak**
Fixes #5110 

### Note

The order of where to pass flags has changed:
before they were put at the end:
```
paster plugin=ckan user add admin password=12345678 email=admin@admin.org -c development.ini
```
now after `ckan`:
```
ckan -c development.ini user add admin password=12345678 email=admin@admin.org
```

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

### CODE REVIEW

**Comment by EvgeniiaVak** [comment]
I removed the classes, extension paster cli commands still work checked with:
```
paster --plugin=ckan datastore set-permissions -c development.ini
```
Also updated user add user (and checked that it works).

Still WIP: 
* investigating if `click_config_option` and `paster_click_group` are needed
* tests failing

**Comment by amercader** [comment]
Great work @EvgeniiaVak ! The test are failing because the Circle CI setup uses the old `paster ` commands to init the database. You'll need to update those to the new version: https://github.com/ckan/ckan/blob/master/.circleci/config.yml#L61:L62

@tino097 @pdelboca this PR includes the changes in #5114 so perhaps we can close that one.
@EvgeniiaVak does the `ckan sysadmin add` command work fine for you in your PR?

**Comment by EvgeniiaVak** [comment]
@amercader 
> does the ckan sysadmin add command work fine for you in your PR?
yes, it works fine.

**Comment by EvgeniiaVak** [comment]
@amercader 
I fixed the settings you've mentioned at https://github.com/ckan/ckan/blob/master/.circleci/config.yml#L61:L62. Some tests still fail - because they were the cli tests, and there were some files importing old commands. I'm working on replacing those.

**Comment by amercader** [comment]
@EvgeniiaVak do you need help to get this over the line? Looking at the failures they seem related to a file already gone from master.

**Comment by EvgeniiaVak** [comment]
@amercader Ok, thanks, I'll update the branch from the master.

**Comment by EvgeniiaVak** [comment]
@amercader 
well, since all the deletions are already in master this issue https://github.com/ckan/ckan/issues/5110 can probably be closed, as it was about the deletions originally.

So this PR is now about adding the tests for ckan cli. I was trying to copy those from paster and remake them for click, and there is still one issue about zero exit code, when there probably shouldn't be one: https://github.com/ckan/ckan/issues/5158.

And also there are already new tests at `ckan/tests/cli/test_cli.py`, so what should I do with this PR:
1. close it, no changes from it are actually needed
2. ✔️add new tests to `ckan/tests/cli/test_cli.py`

File: ckan/tests/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+    def test_cli_user_add_no_args(self):
+        """Command with no args raises SystemExit."""
+        self.user_cmd.args = [u"add"]
+        result = self.runner.invoke(self.user_cmd.add_user)
+        assert result.exception
+        assert u"Missing argument" in result.output
+
+    def test_cli_user_add_no_fullname(self):
+        """Command shouldn't raise SystemExit when fullname arg not present."""
```
Comment (pdelboca) [inline ckan/tests/test_cli.py]:
```suggestion
        """
        Command shouldn't raise SystemExit when fullname arg not present.
        """
```

**Comment by amercader** [comment]
@EvgeniiaVak if you could work on adding the missing tests on `ckan/tests/cli/test_cli.py` That would be great. You can use a new PR or this one, whatever is cleaner or easier for you.

**Comment by EvgeniiaVak** [comment]
@amercader 
reimplementing tests in the new way, I actually created separate files `ckan/tests/cli/test_jobs.py` and `ckan/tests/cli/test_user.py` (with the intention to delete `ckan/tests/test_cli.py`), but with the user tests I have some problems - there are SQL exceptions in the command output:

If I use a new runner - https://github.com/EvgeniiaVak/ckan/blob/fix/remove-paster-cli/ckan/tests/cli/test_user.py#L32-L40 (which I shouldn't probably, no config specified, but it worked for some reason in the [`ckan/tests/test_cli.py`](https://github.com/EvgeniiaVak/ckan/blob/fix/remove-paster-cli/ckan/tests/test_cli.py#L65)), it gives:
```
2020-02-11 20:49:50,041 INFO  [ckan.cli] Using configuration file /Users/evgeniyavakarina/Documents/upwork/datopian/my-forks/ckan/development.ini
2020-02-11 20:49:50,054 INFO  [ckan.config.environment] Loading static files from public
2020-02-11 20:49:50,183 INFO  [ckan.config.environment] Loading templates from /Users/evgeniyavakarina/Documents/upwork/datopian/my-forks/ckan/ckan/templates
2020-02-11 20:49:51,314 INFO  [ckan.config.environment] Loading templates from /Users/evgeniyavakarina/Documents/upwork/datopian/my-forks/ckan/ckan/templates
None - {'email': [u"The email address 'berty@example.com'                     belongs to a registered user."], 'name': [u'That login name is not available.']}
Aborted!
```

And if I use the new set up (as other new tests) - https://github.com/EvgeniiaVak/ckan/blob/fix/remove-paster-cli/ckan/tests/cli/test_user.py#L14-L22, I get:
```
(sqlite3.OperationalError) no such table: user
[SQL: SELECT user.password AS user_password, user.id AS user_id, user.name AS user_name, user.fullname AS user_fullname, user.email AS user_email, user.apikey AS user_apikey, user.created AS user_created, user.reset_key AS user_reset_key, user.about AS user_about, user.activity_streams_email_notifications AS user_activity_streams_email_notifications, user.sysadmin AS user_sysadmin, user.state AS user_state 
FROM user 
WHERE user.name = ? OR user.id = ? ORDER BY user.name
 LIMIT ? OFFSET ?]
[parameters: ('test.ckan.net', 'test.ckan.net', 1, 0)]
(Background on this error at: http://sqlalche.me/e/e3q8)
```

maybe I need to update some settings for it to work?

**Comment by davidread** [comment]
@EvgeniiaVak the tests run for me. Your error suggests you're using sqlite3, which was with the old test.ini. Try running them like this:
```
(default) vagrant@ubuntu-bionic:/usr/lib/ckan/default/src/ckan$ pytest --ckan-ini=test-core.ini ckan/tests/cli/
2020-02-25 20:20:02,527 CRITI [ckan.lib.uploader] Please specify a ckan.storage_path in your config
                         for your uploads
============================================================================== test session starts ===============================================================================
platform linux -- Python 3.6.9, pytest-4.6.5, py-1.8.1, pluggy-0.13.1
rootdir: /usr/lib/ckan/default/src/ckan, inifile: setup.cfg
plugins: split-tests-1.0.9, rerunfailures-8.0, pyfakefs-3.2, cov-2.7.1
collected 39 items

ckan/tests/cli/test_asset.py .                                                                                                                                             [  2%]
ckan/tests/cli/test_cli.py ..........                                                                                                                                      [ 28%]
ckan/tests/cli/test_config_tool.py .....                                                                                                                                   [ 41%]
ckan/tests/cli/test_jobs.py ................                                                                                                                               [ 82%]
ckan/tests/cli/test_search_index.py ..                                                                                                                                     [ 87%]
ckan/tests/cli/test_user.py .....                                                                                                                                          [100%]

=========================================================================== 39 passed in 36.53 seconds ===========================================================================
```

**Comment by davidread** [comment]
@EvgeniiaVak Thanks for all the work on this! I hope you don't mind but I've done a bit of tidying up and created a new PR: #5264

**Comment by EvgeniiaVak** [comment]
Thank you @davidread!

**Pull request closed without merging**